internal/worldstate/db.go

```diff
@@ -56,6 +56,11 @@ type DB interface {
 	// the caller wants from the first key in the database (lexicographic order). An empty
 	// endKey (i.e., "") denotes that the caller wants till the last key in the database (lexicographic order).
 	GetIterator(dbName string, startKey, endKey string) (Iterator, error)
+	// GetSnapshot returns a latest snapshot of the given DB along with all system databases.
```

Comment:
`// GetSnapshot returns a latest snapshot of the given DB along with all system databases`???
From code no snapshot for all system databases created

Reply:
Done. This was the old design but later I changed it to []string so that the consumer can tell which system databases and user databases are needed.